src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -8,73 +9,79 @@ import {
   AccountsRepository,
 } from "@services/mongoose"
 
-export const addInvoiceForSelf = async ({
+export const addInvoice = async ({
   walletId,
   amount,
   memo = "",
-}: AddInvoiceSelfArgs): Promise<LnInvoice | ApplicationError> => {
+}: AddInvoiceArgs): Promise<LnInvoice | ApplicationError> => {
+  const sats = checkedToSats(amount)
+  if (sats instanceof Error) throw sats
```

Comment:
Don't want to throw here. If we start the pattern of using `checkedXxx` methods in the use case functions we need to return the information back to the caller so that GQL can respond back to the user what was wrong.